### PR TITLE
Misc suggestions

### PR DESCRIPTION
**Pull request by Velenir**
Some suggestions
Only `.babelrc` changes are strictly necessary/useful

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr82--dexreact.review.gnosisdev.com

File: src/utils/miscellaneous.ts [hunk 1]
```diff
@@ -1,9 +1,4 @@
 // eslint-disable-next-line
 function noop(..._args: any[]): void {}
 
-// eslint-disable-next-line
-function logImpl(message?: any, ...optionalParams: any[]): void {
-  console.log(message, ...optionalParams)
-}
-
-export const log = process.env.NODE_ENV === 'test' ? noop : logImpl
+export const log = process.env.NODE_ENV === 'test' ? noop : console.log
```
Comment (anxolin) [inline src/utils/miscellaneous.ts]: much cleaner 👍

File: src/components/layout/Header.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ const Wrapper = styled.header`
   color: #ffffff;
   background-color: #3340a9;
   min-height: ${rem('325px')};
-  font-size: 1.5am;
+  font-size: 1.5em;
```
Comment (anxolin) [inline src/components/layout/Header.tsx]: This is messing with the size... 
Compare:
* https://pr82--dexreact.review.gnosisdev.com/
* http://dex.dev.gnosisdev.com

I think we can just remove the line.

Comment (Velenir) [reply src/components/layout/Header.tsx]: removed

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr82--dexreact.review.gnosisdev.com

**Pull request merged**